.github/workflows/ort.yml

```diff
@@ -0,0 +1,207 @@
+
+name: The OSS Review Toolkit (ORT)
+
+on:
+    push:
+        tags:
+            - "ort-for-v*.*"
```

Comment:
I don't think that this is a good trigger. We can either do this on a release tag, or on user action.
magical tags which are used only for side effects don't sound easy to comprehend.